tests/unit/test_app.py

```diff
@@ -25,14 +22,27 @@ def setUp(self):
 
     def tearDown(self):
         m.db.session.remove()
-        m.db.drop_all()
+#        m.db.drop_all()
```

Comment:
Nope, that's a debug-o.  No worries; I'll fix the PR.  We shouldn't merge this until CI passes anyway :)